dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -285,9 +282,8 @@ public Token addGitlabToken(@ApiParam(hidden = true) @Auth User user, @QueryPara
     @Timed
     @UnitOfWork
     @Path("/github")
-    @ApiOperation(value = "Allow satellizer to post a new GitHub token to dockstore", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) },
-        notes = "A post method is required by saetillizer to send the GitHub token",
-        response = Token.class)
+    @ApiOperation(value = "Allow satellizer to post a new GitHub token to dockstore", authorizations = {
```

Comment:
I'd mentioned this in a private Slack, but mentioning satellizer sounds weird to me. Anybody could post a token, not just the Satellizer library, can't they? It seems odd to single out a specific library in our Swagger doc. Not really part of this PR, but seeing it again triggered that thought. :)

Reply:
Hah, was just working on swagger examples in the TRS. 
I don't think there's anything wrong in providing specific examples of how the endpoint is used but it might be worth noting that this can probably be re-used by other libraries.